common/jenkins/README.md

```diff
@@ -0,0 +1,126 @@
+# Creating a Jenkins instance running under Kubernetes on AWS on an EC2 instance
+
+## Inital setup
+1. Create an EC2 instance / start with a Ubuntu Server 18.04 LTS
+1. In the case of AWS choose a sufficiently large EC2 instance (t2.xlarge)
+1. Configure Instance Details - set auto assign public IP
+1. Configure security group to select SSH access for ports 22 and 31750 from your ip address
+1. Configure an existing key pair to access
+1. Click on launch instance
+1. Create an elastic block store volume (80GB) and note down the Volume ID 
+1. Connect to the EC2 instance as follows and select yes:
+```bash
+ssh -i "your pem.pem" ubuntu@ec2-XX-XXX-XXX-XX.eu-west-1.compute.amazonaws.com
+The authenticity of host 'ec2-``XX-XXX-XXX-XX.eu-west-1.compute.amazonaws.com (XX.XXX.XXX.XX)' can't be established.
+ECDSA key fingerprint is SHA256:6W5mciVvBve7QM9HdKtgpqQOgULt/J5So4a/9tuBOCI.
+Are you sure you want to continue connecting (yes/no)? 
+```
+
+
+## Package Installation
+
+### Install microk8s
+MicroK8s is a snap lightweight kubernetes implementation: https://microk8s.io/docs/
+Run the following commands to install microk8s and verify that it's working:
+
+```bash
+ubuntu@ip-XXX-XX-XX-XXX:~$ sudo snap install microk8s --classic
+ubuntu@ip-XXX-XX-XX-XXX:~$ microk8s.kubectl get nodes
+ubuntu@ip-XXX-XX-XX-XXX:~$ microk8s.kubectl get services
+ubuntu@ip-XXX-XX-XX-XXX:~$ sudo snap alias microk8s.kubectl kubectl
+ubuntu@ip-XXX-XX-XX-XXX:~$ kubectl get nodes
+```
+
+Check the status of the microk8s cluster:
+```bash
+ubuntu@ip-XXX-XX-XX-XXX:~$ microk8s.status
+microk8s is running
+addons:
+knative: disabled
+jaeger: disabled
+fluentd: disabled
+gpu: disabled
+storage: disabled
+registry: disabled
+rbac: disabled
+ingress: disabled
+dns: disabled
+metrics-server: disabled
+linkerd: disabled
+prometheus: disabled
+istio: disabled
+dashboard: disabled
+```
+We will now enable the storage and dns add ons:
+
+```bash
+ubuntu@ip-XXX-XX-XX-XXX:~$ microk8s.enable storage
+ubuntu@ip-XXX-XX-XX-XXX:~$ microk8s.enable dns
+
```

Comment:
```suggestion
```

Reply:
enabling the storage and dns components are important otherwise the k8s cluster cannot access external media.

Reply:
agreed the suggestion was just to remove the blank line